src/currencycloud/config.py

```diff
@@ -3,7 +3,12 @@
 from currencycloud.clients.auth import Auth
 import uuid
 import requests
+import os
 
+if os.environ.get('SERVER_SOFTWARE') is not None:
+    if os.environ.get('SERVER_SOFTWARE').startswith(('Development', 'Google App Engine', 'AppScaleServer')):
```

Comment:
This will make the SDK aware of its running environment

Reply:
As discussed in the ticket in JIRA this may not be a route we want to go down.

Additionally with the very recent release of Python 3 on Google App Engine this now become a non-issue. But thats just from my brief investigation on the ticket.